app/views/gsa18f/_proposal_mail.html.haml

```diff
@@ -2,7 +2,8 @@
   %table.w-container.main-container{ width: "800" }
     %tr
       %td
-        %h1.c2_header= "#{title} : #{proposal.name}"
+        %h1.c2_header
+          = link_to proposal.name, proposal_url(proposal),  { target: "C2" }
```

Comment:
no idea. I suspect it was there in order to avoid creating new tabs/windows for each click on an email.